go/master/etcd_client.go

```diff
@@ -142,3 +142,31 @@ func (e *EtcdClient) Load() ([]byte, error) {
 	state := kvs[0].Value
 	return state, nil
 }
+
+// GetKey gets the value by the specify key
```

Comment:
> Subsequent sentences and/or paragraphs can give more details. Sentences should be properly punctuated.

https://github.com/golang/go/wiki/Comments

The comment should be a sentence (ends with ".").